modules/drivers/sqlite/test/metabase/driver/sqlite_test.clj

```diff
@@ -144,7 +144,9 @@
                                    :database-position          0
                                    :database-required          false
                                    :json-unfolding             false
-                                   :database-is-auto-increment false}}}
+                                   :database-is-auto-increment false
+                                   :database-is-nullable       true
+                                   :database-is-generated      false}}}
```

Comment:
Surprised by these - is it just a TODO because of low driver support tier, or was there a technical issue?